test/js/leaves.test.js

```diff
@@ -0,0 +1,46 @@
+import {
+  leafSize,
+  drawLeaf,
+  plusOrMinus,
+  cointoss,
+  randomNumBetween,
+} from "leaves";
+
+describe("randomNumBetween", () => {
+  test("returns number between 0 and specified max", () => {
+    const result = randomNumBetween(10);
+    expect(result).toBeLessThanOrEqual(10);
+    expect(result).toBeGreaterThanOrEqual(0);
+  });
+
+  test("returns number between specified max and specified min", () => {
+    const result = randomNumBetween(5, -5);
+    expect(result).toBeLessThanOrEqual(5);
+    expect(result).toBeGreaterThanOrEqual(-5);
+
+    const result2 = randomNumBetween(10, 5);
+    expect(result2).toBeLessThanOrEqual(10);
+    expect(result2).toBeGreaterThanOrEqual(5);
+  });
+});
+
+describe("plusOrMinus", () => {
+  test("returns number between positive and negative specified value", () => {
+    expect(plusOrMinus(3)).toBeLessThanOrEqual(3);
+    expect(plusOrMinus(10)).toBeGreaterThanOrEqual(-10);
+  });
+});
+
+describe("cointoss", () => {
+  test("returns true or false", () => {
+    expect([true, false]).toContain(cointoss());
+  });
+});
+
+describe("leafSize", () => {
```

Comment:
Do we need any tests for the random distribution of leaves as a whole? Like, "generate 100 leaves and expect at least half to be facing this direction" or so forth – provided we have those expectations!

Reply:
Interesting, that's a helpful idea. I'm not sure we have any expectations like that right now, but I'll keep this in mind.